_data/authors.yml

```diff
@@ -2,6 +2,8 @@ eric:
   name: Eric Carmichael
   email: eric@ckcollab.com
   web: http://ericcarmichael.com
+  avatar_url: ../../../assets/images/avatars/eric1.png
```

Comment:
Hmm, can we do /assets/images/avatars or what-have-you here instead? Some hard static path instead of this?